internal/gengapic/gengrpc.go

```diff
@@ -185,6 +185,9 @@ func (g *generator) grpcClientOptions(serv *descriptorpb.ServiceDescriptorProto,
 	p("    internaloption.WithDefaultAudience(%q),", generateDefaultAudience(host))
 	p("    internaloption.WithDefaultScopes(DefaultAuthScopes()...),")
 	p("    internaloption.EnableJwtWithScope(),")
+	if g.enableNewAuthLibrary() {
```

Comment:
This would become `_, ok := newAuthLibraryAllowlist[g.serviceConfig.GetName()]; ok {`